com.ktechpit.ultimate-media-downloader.yaml

```diff
@@ -0,0 +1,225 @@
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-download:rw
+  - --filesystem=xdg-pictures:rw
+  - --filesystem=xdg-videos:rw
+  - --device=dri
+  - --socket=pulseaudio
+  - --env=XDG_CURRENT_DESKTOP=kde
+  - --env=PACKAGE_TYPE=flatpak
+  
+tags:
+  - proprietary
+  
+modules:
+  - name: lzo
+    config-opts:
+      - --enable-shared
+      - --disable-static
+    cleanup:
+      - /include
+      - /share/doc
+      - '*.la'
+      - /lib/*.so
+    sources:
+      - type: archive
+        url: http://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz
+        sha256: c0f892943208266f9b6543b3ae308fab6284c5c90e627931446fb49b4221a072
+
+
+  - name: squashfs-tools
+    buildsystem: simple
+    build-commands:
+      - XZ_SUPPORT=1 LZO_SUPPORT=1 make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=${FLATPAK_DEST}/bin
+    sources:
+    - type: git
+      url: https://github.com/plougher/squashfs-tools.git
+      tag: '4.5'
+      commit: '0496d7c3de3e09da37ba492081c86159806ebb07'
+      
+      
+  - name: libgumbo
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/google/gumbo-parser/archive/v0.10.1.tar.gz
+        sha256: 28463053d44a5dfbc4b77bcf49c8cee119338ffa636cc17fc3378421d714efad
+    cleanup:
+      - /lib/pkgconfig
+      - /include
+      - '*.a'
+      - '*.la'
+      
+      
+  - name: x264
+    config-opts:
+      - '--disable-cli'
+      - '--enable-shared'
+    sources:
+      - type: git
+        url: https://code.videolan.org/videolan/x264.git
+        commit: 5db6aa6cab1b146e07b60cc1736a01f21da01154
+
+
+  - name: x265
+    buildsystem: cmake-ninja
+    builddir: true
+    subdir: source
+    config-opts:
+      - '-DCMAKE_BUILD_TYPE=Release'
+      - '-DENABLE_CLI=OFF'
+    sources:
+      - type: archive
+        url: https://deb.debian.org/debian/pool/main/x/x265/x265_3.5.orig.tar.gz
+        sha256: e70a3335cacacbba0b3a20ec6fecd6783932288ebc8163ad74bcc9606477cae8
+      
+      
+  - name: libass
+    sources:
+      - type: archive
+        url: https://github.com/libass/libass/releases/download/0.16.0/libass-0.16.0.tar.xz
+        sha256: 5dbde9e22339119cf8eed59eea6c623a0746ef5a90b689e68a090109078e3c08
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/libass/libass/releases/latest
+          version-query: .tag_name
+          url-query: .assets[] | select(.name=="libass-" + $version + ".tar.xz") |
+            .browser_download_url
+      - type: script
+        commands:
+          - autoreconf -fiv
+        dest-filename: autogen.sh
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+      - /share
+      
+      
+  - name: luajit
+    no-autogen: true
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share/man
+    sources:
+      - type: git
+        url: https://luajit.org/git/luajit-2.0.git
+        branch: v2.1
```

Comment:
NEVER a branch. you need an actual commit.

Reply:
Got it. Fixing it.